src/Features/Core/Portable/Snippets/SnippetProviders/AbstractSnippetProvider.cs

```diff
@@ -205,5 +203,25 @@ private async Task<Document> AddFormatAnnotationAsync(Document document, int pos
             document = document.WithSyntaxRoot(annotatedSnippetRoot);
             return document;
         }
+
+        protected virtual SyntaxNode? FindAddedSnippetSyntaxNode(SyntaxNode root, int position, Func<SyntaxNode?, bool> IsCorrectContainer)
```

Comment:
As above:

```suggestion
        protected virtual SyntaxNode? FindAddedSnippetSyntaxNode(SyntaxNode root, int position, Func<SyntaxNode?, bool> isCorrectContainer)
```

Surprised the correctness build didn't fail because of this actually 🤷‍♂️